cmd/promtool/main.go

```diff
@@ -116,8 +116,27 @@ func checkConfig(t cli.Term, filename string) ([]string, error) {
 				return nil, err
 			}
 		}
-	}
 
+		for _, filesd := range scfg.ServiceDiscoveryConfig.FileSDConfigs {
+			for _, file := range filesd.Files {
+				// try to glob file, since it may be like /path/to/files/*.yml
```

Comment:
The comment formatting style in Prometheus projects to use full sentences, always including the trailing dot and to start with capitals. Could you change your comments accordingly?